scala-thoughts.md

```diff
@@ -0,0 +1,16 @@
+
+    What's easy to do in Scala? What's not?
+
+    - I like writing functions in scala actually. They strike a good balance between something like C++ and python in terms of control + ease of definition. So far I havent noticed anything too difficult, but loops seem to be a little wonky, since its mostly a functional program. Once we get into the class definitions im sure that might be better / worse. I think scala is best as a light functional language then the other way around. 
+
+    Sor far, what is/are your favorite language design choice(s) that the designers of Scala made? Why?
+
```

Comment:
I agree, I think the language design relative to other programing languages is a nice median